android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -41,12 +46,18 @@
 
     private static JSONObject initSessionResult = null;
     private BroadcastReceiver mInitSessionEventReceiver = null;
+    private static RNBranchInitListener initListener = null;
```

Comment:
This is  holding  a strong reference to the callback and prevent GC.  Do you think this can lead to a memory  leak. How about keeping this  initListener as a weak reference?